spacetemplate/assets/scrum.yaml

```diff
@@ -0,0 +1,242 @@
+      label: RemainingWork
+      description: TBD
+      type:
+        kind: float
+      required: no
+- id: &productBacklogItemID "eb0829fc-e47e-40c5-9507-a95466e113db"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  name: Product Backlog Item
+  description: TBD
+  icon: fa fa-scissors
+  child_types:
+  - *taskID
+- id: &featureID "a661d2f7-f037-4e66-97ff-f45988791a62"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  name: Feature
+  description: TBD
+  icon: fa fa-puzzle-piece
+  child_types:
+  - *productBacklogItemID
+- id: &bugID "40335566-e207-4231-b95a-f8f5a3b941e6"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  name: Bug
+  description: TBD
+  icon: fa fa-bug
+  child_types:
+  - *productBacklogItemID
+  fields:
+    "system.state": 
+      label: State
+      description: The state of the work item
+      required: yes
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        # First state in this ordered map is implicitly the default state
+        values: 
+        - new
+        - approved
+        - committed
+        - done
+        - removed
+    "system.systeminfo": 
+      label: SystemInfo
+      description: TBD
+      type:
+        kind: asciidoc
+      required: no
+    "system.reprosteps": 
+      label: ReproSteps
+      description: TBD
+      type:
+        kind: asciidoc
+      required: no
+    "system.effort": 
+      label: Effort
+      description: TBD
+      type:
+        kind: float
+      required: no
+    "system.remainingwork": 
+      label: RemainingWork
+      description: TBD
+      type:
+        kind: float
+      required: no
+    "system.acceptancecriteria": 
+      label: AcceptanceCriteria
+      description: TBD
+      type:
+        kind: asciidoc
+      required: no
+    "system.priority": 
+      label: Priority
+      description: TBD
+      type:
+        kind: Integer
+      required: no
+    "system.severity": 
+      label: Severity
+      description: TBD
+      type:
+        kind: Integer
+      required: no
+    "system.foundIn": 
+      label: FoundIn
+      description: TBD
+      type:
+        kind: String
+      required: no
+- id: &epicID "7bd76da5-f5e3-4adf-b976-9f01c9a21416"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  name: Epic
+  description: TBD
+  icon: fa fa-university
+  child_types:
+  - *featureID
+  - *bugID
+  fields:
+    "system.state": 
+      label: State
+      description: The state of the work item
+      required: yes
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        # First state in this ordered map is implicitly the default state
+        values: 
+        - new
+        - in progress
+        - done
+        - removed
+    "system.activatedDate": 
+      label: activatedDate
+      description: TBD
+      type:
+        kind: datetime
+      required: no
+    "system.activatedBy": 
+      label: activatedBy
```

Comment:
Don't abbreviate. I'm not sure if this is maybe a field that should be filled out automatically, don't you think? For that automatic stuff we would the concept of a workflow designer which we don't have at the moment. Therefore such fields cannot go in a space template for now IMHO.